explainer/use_cases.md

```diff
@@ -68,6 +68,17 @@ This use case is the fenced frames with no special restrictions on the src and n
     *   Since there is no unpartitioned/cross-site data available to the fenced frame, it is not able to do any cross-site data joining.
     *   Like all use cases, the fenced frame is isolated from the embedded context via any JS window references, script access, storage access, messaging APIs etc. The fenced frame does not know the embedding site’s origin or etld+1.
 
+To begin experimenting with fenced frames in default mode, enable the following experiments in `chrome://flags`:
+* Privacy Sandbox Ads APIs
```

Comment:
When I search for exactly this text I cannot find the flag you seem to be referencing. Can you put a direct link to it?